packages/main/src/Slider.js

```diff
@@ -82,9 +84,15 @@ class Slider extends SliderBase {
 	constructor() {
 		super();
 		this._stateStorage.value = null;
+		this._setInitialValue("value", null);
 		this.i18nBundle = getI18nBundle("@ui5/webcomponents");
 	}
 
+	onEnterDOM() {
+		this._sliderHandle = this.shadowRoot.querySelector(".ui5-slider-handle");
```

Comment:
Usually we create getters for these and don't cache them.
Additionally, please check the `getFocusDomRef` and `focus` functions in `UI5Element.js`. Basically the idea is that if you mark some DOM element with the `data-sap-focus-ref` attribute, when you (or external code) call `.focus()` on the component, it will automatically focus this DOM element. Just an idea, it's not imperative that you use it, but might make it easier.